tst/unit/test_container_iterator.cpp

```diff
@@ -107,23 +109,21 @@ TEST_CASE("Can pull variables from containers based on Metadata", "[ContainerIte
       ContainerIterator<Real> ci(rc, {Metadata::Independent});
       CellVariableVector<Real> &civ = ci.vars;
       for (int n = 0; n < civ.size(); n++) {
-        ParArrayND<Real> v = civ[n]->data;
+        ParArrayND<Real> &v = civ[n]->data;
```

Comment:
Need to get rid of this '&'

Reply:
hm... wait, is my intuition right here? Now I'm questioning what happens with capturing refs. Aaaah!

Reply:
This is wrong - sorry for the noise.